### DISCUSSION TITLE
Chinese characters are rendered incorrectly when passed through Web STOMP

### DISCUSSION THREAD

**Discussion by wsndng** [rabbitmq/rabbitmq-server#7820] [Other]
### Describe the bug

I am deploying rabbitmq HA in a container environment. When I use the RabbitMQ Web STOMP feature to access message queues through web pages, Chinese characters are garbled

### Reproduction steps

1.It is normal to use the queue in RabbitMQ to send Chinese and obtain Chinese.
2.When accessing message queue information using a. NET client, Chinese is normal.
3.When I use the RabbitMQ Web STOMP feature to access message queues through web pages, Chinese characters are garbled

### Expected behavior

The message was sent in Chinese 123, and the webpage client received garbled code in the image
![微信图片_20230404175511](https://user-images.githubusercontent.com/40561376/229756829-797d4187-1668-41c3-8b73-5cb10982d3e2.png)

**Comment by michaelklishin** [comment]
RabbitMQ does not interpret message contents in any way, they are treated as opaque sequences of bytes.

RabbitMQ management UI uses the encoding selected in browser settings.

Therefore I suspect that the issue is in a particular client or environment that has to render the payload. Please put together a reproducible example because a screenshot does not tell much about what it was and what displays the payload

**Reply by lukebakken** [reply]
@wsndng - I'd like to re-iterate what @michaelklishin said. With a way to reproduce, we can easily fix the issue (if there is one). Right now you are asking that we take a lot of time to _guess_ how to reproduce your issue, which we do not have time to do.

Ideally, you would provide a git repository that we can clone and run (via `docker` or `docker compose`) to see the issue.
